concent_api/common/decorators.py

```diff
@@ -183,6 +184,10 @@ def wrapper(
                 response_from_view = view(request, client_message, client_public_key, *args, **kwargs)
                 if database_name is not None:
                     transaction.savepoint_commit(sid, using=database_name)
+
+            except CreateModelIntegrityError as exception:
+                logger.info(f'CreateModelIntegrityError occurred. View will be retried. Exception: {exception}.')
+                response_from_view = view(request, client_message, client_public_key, *args, **kwargs)
```

Comment:
What will happen if `CreateModelIntegrityError` occurs a second time?

Reply:
This means something went wrong. It can occur only one time, when two or more processes are trying to create the same object. If it happens, one process creates this object, all other fail with `CreateModelIntegrityError`, are restarted, validated one more time and they should't try to create object again (this time it already exists in DB). If `CreateModelIntegrityError` occurs a second time it means that we have bug in our code..